utils/ocm.py

```diff
@@ -134,7 +134,8 @@ def create_cluster(self, name, cluster):
 
         provision_shard_id = cluster_spec.get('provision_shard_id')
         if provision_shard_id:
-            ocm_spec['provision_shard']['id'] = provision_shard_id
+            ocm_spec['properties'] = {}
```

Comment:
I would just verify `ocm_spec['properties']` exists. Just to make sure that someone doesn't fill it in `ocm_spec` and doesn't realize that there's an initialization afterwards

Reply:
done!